pkg/annotation/csv_parser.go

```diff
@@ -51,7 +51,7 @@ func (c *CsvParser) Parse(comment string, annotation Annotation) (e error) {
 	keyValuePair := strings.Split(s, ";")
 	for _, paired := range keyValuePair {
 		keyToValue := strings.Split(paired, ",")
-		if len(keyValuePair) >= 2 {
+		if len(keyToValue) >= 2 {
```

Comment:
This is actually a fun bug that was never noticed. It failed to parse annotations if the annotation only defines one key-value pair :/ both gonut and watchful still define package in the annotation tho which means this was never noticed.

Reply:
This might explain why I struggled to remove the package definition in `gonut` after I read that is not required anymore.